### DISCUSSION TITLE
Using revenue and expense accounts

### DISCUSSION THREAD

**Discussion by M1ck0** [firefly-iii/firefly-iii#9410] [Firefly III] [answered]
I created a revenue account for my paycheck. When creating income I select that revenue account as the source account, and my bank account as the destination account.

When I go to the revenue account, its balance is negative since the transaction was done from revenue account to my bank account.

Is this the correct way to use it? And I guess it's similar for expense accounts too? I just reverse the fields (my bank is source and expense account is destination)

Sorry for kinda spamming the discussion in the last few days.

**Comment by JC5** [comment, marked as answer]
Hi, don't worry about opening issues.

Yes this is correct, and you use the tool correct :)

**Reply by M1ck0** [reply]
Thanks so much for the help≥ 

Love the app <3
